lib/load-config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const path = require('path');
+const fs = require('fs');
+// ESM seems to break `path` and `fs`, require them first before setting it up..
+const requireESM = require('esm')(module);
+const pkgConf = require('pkg-conf');
+
+const loadConfigFile = projectDir => {
+	const loaded = requireESM(path.resolve(projectDir, 'ava.config.js'));
+	const config = (loaded.__esModule) ? loaded.default : loaded;
```

Comment:
Does `esm` take care of this?

Reply:
no, it doesn't seem to. There's a test for this that fails if you don't do the check for `__esModule`.